eventing/event-sources/serverless-sink-source-odc.adoc

```diff
@@ -1,7 +1,7 @@
 :_content-type: ASSEMBLY
 include::_attributes/common-attributes.adoc[]
 [id="serverless-sink-source-odc"]
-= Connecting an event source to an event sink by using the Developer perspective
+= Connecting an event source to an event sink
```

Comment:
Update the topic map with this change.
![image](https://github.com/user-attachments/assets/a9ad6122-30f3-46f8-b879-ee5dfb7b4d8c)